### ISSUE TITLE
[Mate] falls back to xterm instead of using mate-terminal

### ISSUE THREAD

**Issue by oberon2007** [manjaro/desktop-settings#120]
I just noticed that **manjaro-architect** cannot be launched in the Mate live-session because mate tries to launch **xterm**, which isn't installed by default.
Strange thing is that **mate-terminal** is indeed set as the default.
Is this a bug in the Mate Desktop?
The only way I can currently seem to find to launch `setup` with the launcher seems to use
```
terminal=false
Exec=mate-terminal --command -e sudo /usr/bin/setup
```
For the already build 17.1.8 ISO I will just patch this up in the chroot, but somehow it would be good to find a proper solution for this.
Is there a way to export $TERM ? My understanding is that this is kind of a no-go ... @Chrysostomus ?
@Ste74 @philmmanjaro

**Comment by oberon2007** [comment]
Right. So this seems to be an issue upstream is aware of, yet hasn't really managed to fix since 2012, hrrm. I guess we'll have to use some kind of a duct-tape solution with the architect-launcher... :stuck_out_tongue_winking_eye:

**Comment by Ste74** [comment]
Maybe i have forgot this issue .. Good to see you already found it .. :wink:

**Comment by oberon2007** [comment]
For now I have added a workaround in the [desktop-items script](https://github.com/manjaro/manjaro-tools-livecd/blob/879bb9b6b2eafd17ebc38a4d3aec5317c11c9e5a/bin/desktop-items.in#L35).
